Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -92,11 +92,11 @@ def runSelected(opt):
                      11634.911, #2021 DD4hep ttbar reading geometry from XML
                      11634.914, #2021 DDD ttbar reading geometry from the DB
                      11634.0, #2021 ttbar (switching to DD4hep by default)
-                     11634.7, #2021 ttbar mkFit
                      11834.0, #2021 ttbar PU
                      13234.0, #2021 ttbar fastsim
                      13434.0, #2021 ttbar PU fastsim
                      12434.0, #2023 ttbar
+                     12434.7, #2023 ttbar mkFit
```

Comment:
This line is not in the master, right?
https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/scripts/runTheMatrix.py

Reply:
Ah, I forget to look on https://github.com/cms-sw/cmssw/pull/41471. Sorry for the noise.